contrib/clojure-package/examples/infer/imageclassifier/test/infer/imageclassifier_example_test.clj

```diff
@@ -0,0 +1,70 @@
+;; Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
This is great! 👍 

I would really like to see an adapted version of this for the unit tests. I don't think the res-18 is too much to download.
Here is a really fast spike of it to be a better idea of what I mean https://github.com/gigasquid/incubator-mxnet/commit/61cc4b8760fd1a8ef8642aa68b09fed19c4e72a5

Reply:
added